spec/shape.line-spec.js

```diff
@@ -26,6 +26,9 @@ describe("SHAPE LINE", () => {
 						["data3", -150, 120, 110, 140, 115, 125]
 					],
 					type: "line"
```

Comment:
to test `line.step.type`, type should be `step` rather than line.
IMO, create test case for step type is needed, something like below.

And the assertion should be testing to not throw error, because the `step-after/before` is throwing error in previous code.

```js
describe("step type generation", () => {
		before(() => {
			args = {
				data: {
				    columns: [
						["data1", 30, 200, 100, 400,150,250],
						["data2",50,20,10,0,15,25]
					],
                    type: "step"
               }, 
               line: {
                    step: {
                         type: "step-after"
                    }
               }
			};
		});

     it("check line.step.type=step-after option", () => {
           expect(chart).to.not.throw();
     });

    // this is just for changing option value, because beforeEach hook will take this option and generate chart for the next test.
	it("should change to line.step.type option", () => {
		args.line.step.type = "step-before";
		expect(true).to.be.ok;
	});

     it("check line.step.type=step-before option", () => {
           expect(chart).to.not.throw();
     });
};
```